docs/contributing/database-migrations/mssql.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 1
+---
+
+# MSSQL
+
+:::info
+
+For instructions on how to apply database migrations, please refer to the
+[Getting Started](../../getting-started/server/database/mssql/index.md) documentation.
+
+:::
+
+:::tip
+
+We recommend reading [T-SQL Code Style](../code-style/sql.md) since it has a major impact in how we
+write migrations.
+
+:::
+
+## SQL database project
+
+The separate database definitions in `src/Sql/.../dbo` serve as a "master" reference for the
```

Comment:
⛏️ No need for scare quotes here.

```suggestion
The separate database definitions in `src/Sql/.../dbo` serve as the source of truth for the
```

Reply:
It's an analogy to a master copy of something, like an album recording. This content just moved its position by the way.

Reply:
Yes, I know. "The source of truth" is both more inclusive than "master reference", and more accurate as a description.